internal/kubernetes/cert.go

```diff
@@ -0,0 +1,54 @@
+package kubernetes
+
+import (
+	"context"
+
+	"github.com/aws/eks-hybrid/internal/api"
+	"github.com/aws/eks-hybrid/internal/node/hybrid"
+	"github.com/aws/eks-hybrid/internal/validation"
+)
+
+type KubeletCertificateValidator struct {
+	// InstallRoot is optionally the root directory of the installation
+	// If not provided, the installRoot will be empty representing the root (/) of the fs
+	installRoot     string
```

Comment:
what if instead of install root we pass the path to the certificate?
I find that a bit more clear and it avoids path manipulation down the stack